include/ubiquity_motor/motor_parmeters.h

```diff
@@ -83,6 +102,18 @@ struct FirmwareParams {
             nh, "ubiquity_motor/pid_denominator", pid_denominator);
         pid_moving_buffer_size = getParamOrDefault(
             nh, "ubiquity_motor/window_size", pid_moving_buffer_size);
+        controller_board_version = getParamOrDefault(
+            nh, "ubiquity_motor/controller_board_version", controller_board_version);
+        estop_enable = getParamOrDefault(
+            nh, "ubiquity_motor/fw_estop_enable", estop_enable);
```

Comment:
This needs a better name than 'estop enable' maybe like estop_detect?

Reply:
This is in fact an enable of the hardware signal estop logic only available on rev 5.0 boards.  The logic is a bit contorted due to two modes being implemented.   There is a pre rev 5.0 mode that uses the threshold and then there is the signal that shows if switch was setting motor power on or off.    The prefered and not hacky mode is use rev 5.0 signal and the estop_enable register allows us to disable the hardware rev 5.0 mode of estop logic.    I really wish we just wrote off estop on rev 4.9 but sadly we shipped those in units so must kludge it there.

Reply:
I understand what it is, the name is just misleading. 

If i make `estop_enable` false then does the estop button on the front no longer do anything? Obviously not.

A name like `estop_detection` would be make it more clear as to what is actually happening

Reply:
Mark can you take a look at this again

Reply:
Ok, it is pushed.  Tell me if you don't see it.